### PR TITLE
Removes old code and updates build script to reflect changes

### PR DESCRIPTION
**Pull request by bretterer**
## Description:
- Removes quickstart pages
- Removes quickstart-fragments
- Removes vuepress-theme-default (we are using vuepress-theme-prose)
- Updates build scripts for workers
- Moves browser-update.org code to help with `document is not defined` errors

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Review by jakubvul-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js [hunk 1]
```diff
@@ -23,25 +24,20 @@ parentPort.once('message', async (payload) => {
     clientManifest: JSON.parse(payload.clientManifest),
     runInNewContext: false,
     inject: false,
-    shouldPrefetch: siteConfig.shouldPrefetch || (() => false), // changed to false to prevent prefetch. site config was not overwriting this option
+    shouldPrefetch: siteConfig.shouldPrefetch || (() => false),
```
Comment (swiftone) [inline packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js:27]: config.js set this to 'false', but here it is () => false.   which is it?

Comment (bretterer) [reply packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js:27]: `config.js` did not matter what I put there, so it always falls into the `(() => false)`

File: packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
-'use strict'
+`use strict`
```
Comment (swiftone) [inline packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js:1]: ?  You aren't defaulting to all template literals, so why change this one?

File: packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js [hunk 3]
```diff
@@ -54,37 +62,53 @@ module.exports = class Build extends EventEmitter {
...
     const pagePaths = []
-    const pagesPerThread = this.context.pages.length / threads
+    const pagesPerThread = this.context.pages.length / workerThreads
 
-    for (let i = 0; i < threads; i++) {
-      const startIndex = i * pagesPerThread
-      const endIndex = (startIndex + pagesPerThread) > this.context.pages.length ? this.context.pages.length + 1 : startIndex + pagesPerThread
-      const pageData = this.context.pages.slice(startIndex, endIndex)
-      const pages = pageData.map(p => ({ path: p.path, frontmatter: JSON.stringify(p.frontmatter) }))
+    for (let workerNumber = 0; workerNumber < workerThreads; workerNumber++) {
```
Comment (swiftone) [inline packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js:102]: Death to 'i'!

File: packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js [hunk 4]
```diff
@@ -93,36 +117,57 @@ module.exports = class Build extends EventEmitter {
...
+          pagePaths.concat(response.filePaths)
+        }
+        if (response.message) {
+          logger.wait(response.message)
+        }
       })
-      worker.on('error', (error) => {
+      worker.on('error', error => {
+        // readline.cursorTo(process.stdout, 0, i)
+        // readline.clearLine(process.stdout, 0)
```
Comment (swiftone) [inline packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js:137]: Do these still apply?

**Review by swiftone — APPROVED** [review]

File: packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js (hunk 4 above)
Comment (bretterer) [reply packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js:137]: Nope, removed

File: packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js (hunk 2 above)
Comment (bretterer) [reply packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js:1]: Changed

File: packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js (hunk 1 above)
Comment (bretterer) [reply packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js:27]: I removed prefetch from the config since its ignored it seems

**Pull request merged**